javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+//let width
+//let depth
+//let height
+//let volumeInMeters = width * depth * height
+//function calculateHousePrice(volumeInMeters, gardenSizeInM2) {
+//  return
+//}
```

Comment:
It can be removed, if you want to put some comments as explanations, it should be meaningfully